Backpack-SwiftUI/Calendar/Classes/Core/CalendarContainer.swift

```diff
@@ -92,16 +136,65 @@ struct CalendarContainer_Previews: PreviewProvider {
         let end = calendar.date(from: .init(year: 2025, month: 12, day: 25))!
         let monthScroll = MonthScroll(monthToScroll: start)
 
-        CalendarContainer(
-            calendar: calendar,
-            validRange: start...end,
-            monthScroll: monthScroll,
-            monthContent: { monthNumber in
-                VStack {
-                    BPKText("Calendar Grid \(monthNumber)")
-                    Divider()
+        GeometryReader { proxy in
+            CalendarContainer(
+                calendar: calendar,
+                validRange: start...end,
+                parentProxy: proxy,
+                monthScroll: monthScroll,
+                onScrollToMonth: nil,
+                monthContent: { monthNumber in
+                    VStack {
+                        BPKText("Calendar Grid \(monthNumber)")
+                        Divider()
+                    }
+                }
+            )
+        }
+    }
+}
+
+// MARK: - Item Visibility Detection in Scroll View
+private struct ItemVisibilityPreferenceKey: PreferenceKey {
```

Comment:
it's not critical, but ItemVisibilityPreferenceKey and ItemVisibilityObserver could be in another file, so not to not make this file grow too much